README.md

```diff
@@ -45,7 +45,15 @@ See also [rules_docker](https://github.com/bazelbuild/rules_docker) for a simila
 
 See the [Frequently Asked Questions (FAQ) wiki page](/../../wiki/Frequently-Asked-Questions-(FAQ)).
 
-## Community
+## Get involved with the community
 
+We welcome contributions! Here's how you can contribute:
+
+* Browse [issues](https://github.com/GoogleContainerTools/jib/issues) or file [an issue](https://github.com/GoogleContainerTools/jib/issues/new)
```

Comment:
Seems a bit better to include the verb in the link? _Browse issues_ or _file an issue_. What do you think?

Reply:
Sounds good!